spec/features/neighborhoods/admin_creates_new_neighborhood_spec.rb

```diff
@@ -15,11 +15,15 @@
     fill_in "Name", with: "Downtown"
     select("East", from: "Location")
     fill_in "Description", with: "Lame."
+    attach_file("neighborhood_image", "spec/fixtures/Stonehenge.jpg")
     click_on "Submit"
 
+    neighborhood = Neighborhood.last
+
     expect(page).to have_content("Neighborhood Added!")
-    expect(page).to have_content("Downtown")
-    expect(page).to have_content("EAST")
+    expect(page).to have_content(neighborhood.name)
```

Comment:
I think we should leave Downtown and EAST in the expect in case something goes wrong in the save.